bibcheck/plugins/fermilab_coll

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of INSPIRE.
+## Copyright (C) 2018 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""
+Add 980__a:Fermilab to HEP records with a Fermilab report number
+"""
+
+def check_records(records):
+    for record in records:
```

Comment:
I know I said this will work, however I have some concerns:

Looking at it now, this plugin has no failsafe, it will blindly change all records it is called on, it relies on the `rule` to do proper filtering, which makes it dangerous. What if someone who is not aware of this, run this plugin on all records -- maybe as part of a general run of all plugins? 
So I suggest to add a check that there is indeed some `Fermilab` in `037a/z`
before the `record.add_field`, and also that there is no `980:Fermilab` already, just to be on the safe side.